ooze_mutation_config_test.go

```
@@ -12,118 +11,69 @@ import (
-		"campaign-runner":    {"./internal/ooze"},
-		"campaign-cycle":     {"./internal/ooze"},
-		"campaign-emergency": {"./internal/ooze"},
-		"campaign-effects":   {"./internal/ooze"},
-		"darwin":             {"./internal/ooze"},
-	}
-	for _, shard := range selfMutationProductionShards {
-		assert.Equal(t, want[shard.name], shard.packages, "mutation shard %q packages=%#v, want %#v", shard.name, shard.packages, want[shard.name])
-		for _, packagePattern := range shard.packages {
-			assert.NotEqual(t, "./...", packagePattern, "mutation shard %q repeats the full module test suite", shard.name)
-		}
+var selfMutationPortableProductionPaths = []string{
```

golangci-lint (2.12.2)

[error] selfMutationPortableProductionPaths is a global variable

(gochecknoglobals)